Autopilot/Inc/TelemPathInterface.hpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * Provides declarations common to the telemetry and path manager interfaces.
+ * Author: Anthony Bertnyk, Messed with by Gordon Fountain to turn from Attitude Manager to Telemetry
+ */
+
+#ifndef TELEM_PATH_INTERFACE_HPP
+#define TELEM_PATH_INTERFACE_HPP
+
+//Data for path manager to send to telemetry.
+typedef struct DataForTelem{
+	int errorCode; //Code for specified errors
+	float gpsLattitude,gpsLongitude; //Current gps position
+	int curAltitude,curAirspeed; //Airspeed and altitude data
+	float roll,pitch,yaw;	//Current orientation (radians)
+	bool isLanded; //Switch to check if landed (maybe limit switch?)
+} DataForTelem;
+
+//Data for telemetry to send to path manager.
+typedef struct TelemToPMData{
```

Comment:
Again, the name TelemToPmData is inconsistent with the name CommandsForPM used elsewhere.